### DISCUSSION TITLE
Hierarchical (linked) flimant profiles?

### DISCUSSION THREAD

**Discussion by NameOfTheDragon** [supermerill/SuperSlicer#4323] [Q&A]
Is it possible to create linked filament profiles? Let me explain.

I find that I create multiple profiles for a type of filament which have a lot in common.. For example,, I might create a profile for "eSun ABS" and then I'd have a bunch of derivative profiles that vary in colour and perhaps extrusion multiplier. I might also create profiles with different volumetric flow rates for use with different nozzles.

Then one day I discover that I get better results with slightly different cooling parameters. Now I have to go through every profile and change that setting, and its easy to make mistakes. What I really need to do is only change my generic "eSun ABS" profile and have that automatically flow through to the derived profiles.

I have a strong hunch that SS can do this, but it isn't obvious to me how. Is my hunch correct?

**Comment by kainz** [comment]
Yes there is, but its not easily user accessible right now -- in your SuperSlicer settings directory, each profile is written as an ini file, you can add a setting `inherits = PROFILE` where PROFILE is the setting you wish to inherit from (you will then need to restart superslicer to see the inherited profiles take effect).

This is a fairly longstanding issue, see https://github.com/supermerill/SuperSlicer/issues/4232 and its parent issues in PrusaSlicer.  The UI isnt great for it, but by editing the inifilles on disk, you can do this.

**Reply by kainz** [reply]
(im also not sure linked profile changes actually propagate more than 1 profile 'deep'
